packages/@webex/plugin-cc/src/services/core/Utils.ts

```diff
@@ -12,9 +12,16 @@ const getCommonErrorDetails = (errObj: WebexRequestPayload) => {
 
 export const getErrorDetails = (error: any, methodName: string) => {
   const failure = error.details as Failure;
-  LoggerProxy.logger.error(`${methodName} failed with trackingId: ${failure?.trackingId}`);
+  const reason = failure?.data?.reason ?? `Error while performing ${methodName}`;
+  // We don't want to log silentReLogin errors
+  if (methodName !== 'silentReLogin') {
```

Comment:
You're correct, I think it will be good if I add another check which also checks the type - ie AGENT_NOT_FOUND and silentRelogin, this way we know it will be only the case we want silenced.

Reply:
You're correct, I think it will be good if I add another check which also checks the type - ie AGENT_NOT_FOUND and silentRelogin, this way we know it will be only the case we want silenced.